data/tlg0535/tlg001/tlg0535.tlg001.perseus-eng2.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title>On the Twelve Years </title>
+                <author>Demades</author> 
+                <editor role="translator">J. O. Burtt</editor>
```

Comment:
Only comment is that I thought we were starting to spell out all the editor and translator names, so if that is the case, this should be: John Ormiston Burtt

Reply:
I think I could not find it. Thanks.

Reply:
So I was looking at the last file, I will now go approve.